internal/crq/config_test.go

```
@@ -27,6 +27,33 @@ func TestBuildConfigRetainsACopyOfTheInputEnvironment(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "env")
+	if err := os.WriteFile(path, []byte("CRQ_WORK_OWNER=file-session\n"), 0o600); err != nil {
+		t.Fatal(err)
+	}
+	t.Setenv("CRQ_CONFIG", path)
+	old, had := os.LookupEnv("CRQ_WORK_OWNER")
+	if err := os.Unsetenv("CRQ_WORK_OWNER"); err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		if had {
+			os.Setenv("CRQ_WORK_OWNER", old)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)